### PR TITLE
feat: add sealed secrets

### PR DESCRIPTION
**Pull request by j-zimnowoda**
### Implements:
https://app.zenhub.com/workspaces/dev-5e7e84dddc966f05a627a19b/issues/gh/redkubes/otomi-core/1441
https://github.com/bitnami-labs/sealed-secrets

### Description:
This PR adds a feature to create sealed secrets.
Is paired with: 
https://github.com/redkubes/otomi-api/pull/500
https://github.com/redkubes/otomi-console/pull/381
https://github.com/redkubes/redkubes.github.io/pull/53

## Checklist

- [ ] Architecture Design Records have been added as `adr/*.md` and appended to list in `adr/_index.md`, if applicable.
- [ ] The `values-schema.yaml` file and `test/**` fixtures have been updated to reflect code changes, if applicable.
- [ ] The OpenApi Schema from redkubes/otomi-api project is compatible with definitions from `values-schema.yaml` file, if applicable.
- [ ] Helm releases are meeting otomi's baseline security policies, if applicable.
- [ ] Helm chart and helmfile changes are tested against upgrade scenario, if applicable.

### CODE REVIEW

File: apps.yaml [hunk 1]
```diff
@@ -304,6 +304,16 @@ appsInfo:
     license: Apache 2.0
     about: Prometheus is a systems and service monitoring system. It collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts when specified conditions are observed.
     integration: Prometheus can be activated to aggregate all platform metrics and store them in a storage endpoint of choice (defaults to PVC). When Otomi is configured in multi-tenancy mode, each team will be provided with a dedicated Prometheus instance. This instance can be used to aggregate custom team metrics.
+  sealed-secrets:
```
Comment (j-zimnowoda) [inline apps.yaml:307]: Could you please fill in this section?

File: src/dev/addApp.ts
Comment (j-zimnowoda) [inline src/dev/addApp.ts:1]: This file can be removed as @ElderMatt has newer version in his PR.

File: tools/Dockerfile
Comment (j-zimnowoda) [inline tools/Dockerfile:1]: since you develop JS code to encrypt/decrypt data then we do not need to add kubeseal

**Review by ElderMatt — APPROVED** [review]
LGTM

**Pull request merged**